Python/peephole.c

```diff
@@ -255,7 +255,7 @@ PyCode_Optimize(PyObject *code, PyObject* consts, PyObject *names,
        than +255 (encoded as multiple bytes), just to keep the peephole optimizer
        simple. The optimizer leaves line number deltas unchanged. */
 
-    for (j = 0; j < tabsiz; j += 2) {
+    for (j = 0; j < (unsigned int) tabsiz; j += 2) {
```

Comment:
This would just sweep the issue under cover. If tabsize larger than maximal `unsigned int` the code will not work.

I think it will be more robust and consistent to replace the loop variable `j` with `i` as in the other loop below.